public/js/add_video.js

```diff
@@ -1,7 +1,9 @@
+var index = 0;
 const addTagFormDeleteLink = (tagFormLi) => {
     const removeFormButton = document.createElement('button')
     removeFormButton.classList
-    removeFormButton.innerText = 'Delete this tag'
+    removeFormButton.innerText = 'Supprimer ce lien'
+    removeFormButton.setAttribute('class','btn-remove-video btn-secondary' )
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/soutenance14/ST/pullRequest?prid=8212089)
```suggestion
    removeFormButton.setAttribute("class",'btn-remove-video btn-secondary' )
```